### DISCUSSION TITLE
Invoke an external API

### DISCUSSION THREAD

**Discussion by PraveenSuryaNarayanaMoorthy** [apideck-libraries/portman#655] [Q&A] [answered]
I've a scenario where I need to call an external API (s3 url) which is generated as a response of one of a POST call and after that need to execute another call. In short I need to execute in below order

POST call (response have a URL) --> execute that URL (response have an id and array) ---> Another POST call (passing the response from previous call)

But when checked the docs, i'm not sure how to call an external URL.

**Comment by thim81** [comment, marked as answer]
hi @PraveenSuryaNarayanaMoorthy 

You could use the "Pre-request" functionality of Postman to trigger a request.
Have a look at this [GIST](https://gist.github.com/thim81/a1e218272fbcf934bcbc2d19cab70fef) with a Pre-request script.

<img width="1361" alt="image" src="https://github.com/user-attachments/assets/1aacee03-281d-4a8e-8885-4f6145bc1254">
